packages/server/lib/project-base.ts

```diff
@@ -83,18 +83,18 @@ export class ProjectBase<TServer extends ServerE2E | ServerCt> extends EE {
   private _recordTests?: any = null
 
   public browser: any
-  public projectType: RunnerType
+  public testingType: RunnerType
```

Comment:
Why not rename the type as well here? Too much work?

Reply:
Good idea, done. I just did find and replace on `testingType` - that's why I missed `RunnerType`. It's now using `Cypress.TestingType`.